README.md

```diff
@@ -19,23 +19,27 @@ Make sure that:
 ### Check your SSH connection to all servers
 
 ```
```

Comment:
shell?

Reply:
Intentionally omitted because what follows is documentation not a list of commands that can be copied/pasted.